packages/office-ui-fabric-react/src/components/Button/sass/ButtonCompound.scss

```diff
@@ -0,0 +1,83 @@
+@import '../../../common/common';
+
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
```

Comment:
Minor suggestion; if we have a HeroButton and a CompoundButton, we should name these files the same as the classname. (E.g. CompoundButton.scss)